pycbc/results/versioning.py

```diff
@@ -21,72 +21,100 @@
 import subprocess
 from pycbc.results import save_fig_with_metadata, html_escape
 
+import lal, lalframe, lalmetaio, lalinspiral, lalsimulation
+import glue.git_version, pycbc.version
+
 def get_library_version_info():
     """This will return a list of dictionaries containing versioning
     information about the various LIGO libraries that PyCBC will use in an
     analysis run."""
     library_list = []
-    import lal, lalframe, lalmetaio, lalinspiral, lalsimulation
-    import glue.git_version, pycbc.version
+
+    def add_info_new_version(info_dct, curr_module, extra_str):
+        crm = curr_module
+        exs = extra_str
+        info_dct['ID'] = eval(crm + '.' + exs + 'VCSInfo.vcsId')
```

Comment:
@spxiwh You can do 

vcs = getattr(crm, 'VCSInfo')
info_dct['ID''] = vcs.vcsId